src/constants.js

```diff
@@ -42,3 +42,5 @@ export const MESSAGE_FETCH_START = 'MESSAGE_FETCH_START';
 export const MESSAGE_FETCH_SUCCESS = 'MESSAGE_FETCH_SUCCESS';
 
 export const PRESENCE_RESPONSE = 'PRESENCE_RESPONSE';
+
+export const MESSAGE_FLAG_STARRED = 'starred';
```

Comment:
This file is exclusively for redux action constants.
Though, I guess it is absolutely not apparent from the naming.